.github/workflows/buid_and_release.yml

```diff
@@ -39,3 +39,4 @@ jobs:
           output/release/medgen.obo
           output/release/medgen-disease-extract.obo
           output/release/medgen-xrefs.robot.template.tsv
+          output/release/medgen.sssom.tsv
```

Comment:
## New output(s)
@matentzn I did 1 output. Do you think this ought to be 1 `medgen.sssom.tsv` or should it be 2, like `medgen-hpo.sssom.tsv` and `medgen-hpo.sssom.tsv`?